### DISCUSSION TITLE
Can I shut Off Logging

### DISCUSSION THREAD

**Discussion by ShaunRoberts543** [iRayanKhan/homebridge-tuya#354] [New device help]
Hello, I have a "simple heater" set-up and it functions ok, but it is blowing up my logs.  What can I add to configuration to eliminate this?

**Comment by VCTGomes** [comment]
Omg! I need this too. The log from Tuya-Debug are terrible!

I was needing to see something happens this morning and I can't because the Tuya-Debug are filling up all my log!

**Comment by dibsies** [comment]
Same here. It seems like this wouldn't be a very difficult thing to implement. Has this plugin been abandoned?

**Comment by dibsies** [comment]
More info here: https://github.com/iRayanKhan/homebridge-tuya/issues/265

It sounds like they were working on adding a feature to disable logging before development stalled.

**Comment by 05TEVE** [comment]
I have merged in 2 PRs as part of the 3.1.0 release i am working on that push the logging into the Homebridge logging itself. 
This should quiet the logging down significantly. The latest beta on homebridge has the changes includes so should be good for testing.
